### PR TITLE
Switch to use indexhtml-webpack-plugin instead of the custom html-plu…

### PR DESCRIPTION
**Pull request by nkronlage**
I stumbled on indexhtml-webpack-plugin while looking for a way to use html as an entry point and then found this really nice webpack-single-page-boilerplate project which supports babel so I was hoping to combine the two.  Here's a change to remove the custom plug in and use the indexhtml-webpack-plugin.

### CODE REVIEW

**Comment by nihey** [comment]
Thanks!

I'll check it out as soon as I get home, if everything is good, it'll be merged.

There was just a small `eslint` error on `webpack.config.js`, I commented it on the code.
It would be good if you can get it fixed untill then (about 23:00 UTC) - but if you can't, don't worry, I can do it too.

File: webpack.config.js [hunk 1]
```diff
@@ -17,18 +19,22 @@ module.exports = {
...
+        loader: cssExtractTextPlugin.extract('style-loader', 'css-loader'),
       },
       {
         test: /\.less$/,
-        loader: ExtractTextPlugin.extract('style-loader', 'css-loader!less-loader'),
+        loader: cssExtractTextPlugin.extract('style-loader', 'css-loader!less-loader'),
+      },
+      {
+        test: /\.html$/,
+        loader: 'html?attrs=link:href img:src'
```
Comment (nihey) [inline webpack.config.js]: It seems the tests failed because there was a missing comma on the end of this line.

**Comment by nkronlage** [comment]
Thanks for looking at this.  Added the missing comma.

**Comment by nihey** [comment]
Worked Wonderfully, thanks @nkronlage . I'll merge it and update the `README.md`

**Pull request merged**